example/android/build.gradle

```diff
@@ -12,3 +12,14 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
     }
 }
+
+allprojects {
+    repositories {
+        maven {
+            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
```

Comment:
Nit, Since this comment is taken from the upstream issue, is it worth referencing somewhere?

Reply:
Oh, never mind then!